vgnode.go

```diff
@@ -32,7 +32,7 @@ const (
 
 // VGAttribute is the attribute on an HTML tag.
 type VGAttribute struct {
-	Namespace, Key, Val string
+	Namespace, NamespaceURI, Key, Val string
```

Comment:
This concerns me - VGAttribute is copied from https://godoc.org/golang.org/x/net/html#Attribute and I want to consider carefully before changing it.

If the NamespaceURI can be derived from `namespaceToURI`, why do we need to store it?